src/encoder.rs

```diff
@@ -1207,16 +1207,17 @@ pub fn encode_block_b(fi: &FrameInvariants, fs: &mut FrameState,
 
             luma_mode.predict_inter(fi, p, &po, &mut rec.mut_slice(&po), plane_bsize);
         }
-        write_tx_tree(fi, fs, cw, w, luma_mode, bo, bsize, tx_size, tx_type, skip, bit_depth); // i.e. var-tx if inter mode
+        write_tx_tree(fi, fs, cw, w, luma_mode, bo, bsize, tx_size, tx_type, skip, bit_depth, false); // i.e. var-tx if inter mode
```

Comment:
Should this also be false in the inter case? The compute_tx_type() function in the spec returns the same tx type for luma and chroma when is_inter is true.

Reply:
You mean true (that is luma_only), that is already done, but not here and inside tx_type_decision() in rdo.rs (Line 311).

Reply:
Sorry, commented on the wrong part of the code...